### PR TITLE
Exposing Concourse Metrics for Prometheus

### PR DESCRIPTION
**Pull request by minniux**
Expose metrics for Prometheus on `0.0.0.0:9391` and allow traffic to and from it 

Related issue: https://github.com/skyscrapers/engineering/issues/25

### CODE REVIEW

File: ecs-web/elb.tf [hunk 1]
```diff
@@ -49,3 +49,23 @@ resource "aws_security_group_rule" "sg_ecs_instances_elb_out_ssh" {
...
+}
+
+# Allow trrafic from Prometheus metrics to the ECS security group
+resource "aws_security_group_rule" "sg_ecs_instances_in_metrics" {
+  security_group_id = "${var.backend_security_group_id}"
+  type              = "ingress"
+  from_port         = "${var.concourse_prometheus_bind_port}"
+  to_port           = "${var.concourse_prometheus_bind_port}"
+  protocol          = "tcp"
+  cidr_blocks       = ["0.0.0.0/0"]
```
Comment (SamClinckspoor) [inline ecs-web/elb.tf]: doesn't this expose the metrics to the world?

Comment (minniux) [reply ecs-web/elb.tf]: done

Comment (SamClinckspoor) [reply ecs-web/elb.tf]: I think we need some more restrictions here. I don't think we want the metrics to be publicly available, but only from  our own prometheus.

File: ecs-web/elb.tf [hunk 2]
```diff
@@ -49,3 +49,23 @@ resource "aws_security_group_rule" "sg_ecs_instances_elb_out_ssh" {
   protocol          = "tcp"
   cidr_blocks       = ["0.0.0.0/0"]
 }
+
+# Allow traffic to Prometheus metrics from the ECS security group
+resource "aws_security_group_rule" "sg_ecs_instances_out_metrics" {
```
Comment (iuriaranda) [inline ecs-web/elb.tf]: I think you need to allow access to the prometheus port from the ELB no? As prometheus works by polling its targets, it'll be prometheus that makes requests to the ELB, and the ELB to the backend

**Comment by venturel** [comment]
New PR created and Already merged closing this

**Pull request closed without merging**